grakn-engine/src/main/java/ai/grakn/engine/controller/SystemController.java

```diff
@@ -71,10 +76,45 @@
     private final Logger LOG = LoggerFactory.getLogger(SystemController.class);
     private final EngineGraknGraphFactory factory;
 
-    public SystemController(EngineGraknGraphFactory factory, Service spark) {
+    public SystemController(EngineGraknGraphFactory factory , Service spark) {
         this.factory = factory;
-        spark.get(KEYSPACES,     this::getKeyspaces);
-        spark.get(CONFIGURATION, this::getConfiguration);
+
+        spark.get(INITIALISE,         this::initialiseSession);
```

Comment:
I'm not sure. I feel that we should be separating the functionality between endpoints that modify the backend and endpoints that only get data. It feels a bit odd to be writing to the graph based on the presence of an optional parameter.

Writing this, I realised that the `INITIALISE` endpoint should be a `POST` no matter what, because it does modify the graph.

Ideally I would like `INITIALISE` to only add the keyspace to the system graph and not return the properties. But I realise that would mean that graph needs to make two API calls, which is also not something we want.